spec/features/history/internal_note_spec.rb

```diff
@@ -25,8 +25,10 @@ def and_i_fill_in_and_submit_a_internal_note
   end
 
   def then_i_should_see_the_internal_note
-    expect(page).to have_content(I18n.t!("documents.history.entry_types.internal_note"))
-                .and have_content(@edition.timeline_entries.last.details.body)
+    within(".app-timeline-entry.app-timeline-entry--highlighted") do
+      expect(page).to have_content(I18n.t!("documents.history.entry_types.internal_note"))
+                  .and have_content(@edition.timeline_entries.last.details.body)
+    end
```

Comment:
I don't think we should use the feature tests to assert the highlighting. Our convention is to use the within option of feature tests just to distinguish particular content from others. I think using the within in this way is somewhat cryptic that it is actually a test. For example if it fails a developer may think "oh the within is just on the wrong selector".

I think if we actually want to test this logic we'd be better placed doing this by testing the particular view. In that case we can then that given this particular input we expect this particular selector. Bruce is working in a similar area in https://github.com/alphagov/content-publisher/pull/1644 by the way.

Reply:
I'm still trying to get the grasp of what's the best way to test the presence of a class on a specific element with capybara then where to place this test functional/view, or both. Will catch-up with Bruce.

Reply:
Does this doc help at all: https://github.com/alphagov/content-publisher/blob/master/docs/testing-strategy.md ? (I really hope it does 🙏 )

Reply:
That helped, thank you. I rebased this on #1644 since there are some changes in the way views are being structured coming up. #1644 introduces views tests for timeline entries and whitehall imported entries, but there's still no test for document history overall – which is where this assertion would need to be. I'm happy to create it, but I will need some support in creating a mock document with timeline entries for this.